data/json/monsters/feral_humans.json

```diff
@@ -215,5 +215,135 @@
     "special_attacks": [ [ "TAZER", 10 ] ],
     "luminance": 500,
     "death_drops": "feral_security_death_drops_flashlight"
+  },
+  {
+    "id": "mon_feral_prepper",
+    "type": "MONSTER",
+    "name": { "str": "feral prepper" },
+    "description": "A quick glance of this person would show that they have prepared well for times like these.  Unfortunately, their bloodshot eyes, and the excluding treatment by the undead shows that they have turned feral.  One of their hands clasp the handle of their prepped improvised weapon, while the other clasps unto the grip of a handgun, treating both as clubs.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 75,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_cut": 5,
+    "dodge": 4,
+    "armor_bash": 7,
+    "armor_cut": 7,
+    "armor_bullet": 6,
+    "armor_acid": 4,
+    "vision_day": 50,
+    "vision_night": 3,
+    "harvest": "human",
+    "upgrades": { "into": "mon_feral_survivalist", "half_life": 60 },
+    "starting_ammo": { "9mm": 3 },
+    "special_attacks": [
```

Comment:
```
    "special_attacks": [
      {
        "type": "gun",
        "cooldown": 10,
        "move_cost": 150,
        "gun_type": "m9",
        "ammo_type": "9mm",
        "fake_skills": [ [ "gun", 1 ], [ "handgun", 2 ] ],
        "fake_dex": 8,
        "fake_per": 10,
        "ranges": [ [ 0, 14, "DEFAULT" ] ],
        "require_targeting_player": false,
        "description": "The feral security guard fires their Beretta M9A1!"
      }
    ],
```

This is the special attack for the feral security guard.  In current play once the security guard gets into range he will shoot without missing until he runs out of ammo.  I think this means there is an issue in gun actor causing monster special attacks to almost never miss. Since a PC with these abilities should miss some times.

Reply:
While experimenting how much damage the feral prepper does to a early character (I should also test the feral survivalist as well after that), I spawned the feral prepper a bit closer than usual when I test their dispersion, and their second shot managed to hit me in the arm. Thing is, their third shot went pass me, even though they were more closer that their second shot in distance. So maybe, The feral 9mm guard could also (maybe) copy the fake gun from the feral survivalist. It would make sense, they are both in context "experienced" with their handgun. The dispersion of the monster_gun, i think i named it feral_experienced_gun has a dispersion of 1500, which for me sounds reasonable, they start hitting their shots when they are 2-3/4-5 total distance near the PC, but as seen with the feral prepper, they can miss their shot again although they're closer than before. But if you don't like the dispersion amount, we could increase it again more than 1500? What do think?

Reply:
out of 7 bullets (supposed to be 5 total, but i was using different device with outdated files), 5-6 bullets missed PC but a guarantee 1 bullet hit when feral spot and shot at total feral's sight distance. Should I decrease dispersion?

Reply:
Nevermind, with 6 bullets, 1500 dispersion is a good (enough) design. Atleast 3 bullets would likely hit, which is alright/enough.